### ISSUE TITLE
Clicking on the list does not select the item

### ISSUE THREAD

**Issue by ranchmed** [devbridge/jQuery-Autocomplete#809]
Clicking an item on the selection list sometimes does not select the item. You need to click fast to select the item. If you are slow releasing the mouse button on the click, the item is not selected.

I noticed that this problem does not exist on the demo website. How can I fix this so that the item is selected regardless of how fast the mouse button is released during a click operatoin?

**Comment by boldtrn** [comment]
Good catch, I was hunting this issue for months! This behavior was introduces here: https://github.com/devbridge/jQuery-Autocomplete/commit/d556a873e7fa6138e4f630d3859f9aed9b975a29

Maybe we could increase 200 ms to 300-400 ms?

**Comment by boldtrn** [comment]
BTW: the demo website uses a very old version of the autocomplete, you might not want to check that too closely ;)

**Comment by tkirda** [comment]
Closing as aged. This was filed against a pre-2.0 codebase that has since been substantially reworked — the source is now TypeScript, the build and CI pipeline are different, and a number of relevant code paths around positioning, selection, and event flow have changed in `2.0.1`.

If this still reproduces against current `master`, please open a new issue with the autocomplete options used and a short steps-to-reproduce.
